plugins/quota/rest.go

```diff
@@ -115,9 +130,77 @@ func quotaEnableHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func quotaDisableHandler(w http.ResponseWriter, r *http.Request) {
-	// implement the help logic and send response back as below
+	// Collect inputs from URL
+	p := mux.Vars(r)
+	volName := p["volname"]
+
 	ctx := r.Context()
-	restutils.SendHTTPResponse(ctx, w, http.StatusOK, "todo: quota disable")
+	logger := gdctx.GetReqLogger(ctx)
+
+	// Validate volume existence
+	vol, err := volume.GetVolume(volName)
+	if err != nil {
+		restutils.SendHTTPError(ctx, w, http.StatusNotFound, errors.ErrVolNotFound.Error(), api.ErrCodeDefault)
```

Comment:
@harigowtham api.ErrCodeDefault is no more needed, Please see changes made in https://github.com/gluster/glusterd2/pull/613/files

Reply:
done